core/src/test/java/eu/fasten/core/vulchains/VulnerableCallChainTest.java

```diff
@@ -0,0 +1,50 @@
+package eu.fasten.core.vulchains;
+
+import eu.fasten.core.data.FastenURI;
+import eu.fasten.core.data.vulnerability.Vulnerability;
+import java.util.LinkedList;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class VulnerableCallChainTest {
+
+    private static VulnerableCallChain sut1;
+    private static VulnerableCallChain sut;
+    private static VulnerableCallChain sut1Prime;
+
+    @BeforeAll
+    public static void setUp() {
+        sut = new VulnerableCallChain(new LinkedList<>(), new LinkedList<>());
+        sut1 = new VulnerableCallChain(List.of(new Vulnerability("1234")),
+            List.of(FastenURI.create("fasten://mvn!g:a$1.0.0/x/C.m()%2Fjava.lang%2FVoidType")));
+        sut1Prime = new VulnerableCallChain(List.of(new Vulnerability("1234")),
+            List.of(FastenURI.create("fasten://mvn!g:a$1.0.0/x/C.m()%2Fjava.lang%2FVoidType")));
+    }
+
+    @Test
+    void testEquals() {
+        Assertions.assertNotEquals(sut, sut1);
```

Comment:
Maybe it'd be a good idea to create 3 separate tests for the 3 assertions. I'll leave this up to you.